packages/components/src/internal/renderers.tsx

```diff
@@ -94,6 +169,19 @@ export const HeaderCellDropdown: FC<HeaderCellDropdownProps> = memo(props => {
         handleAddColumn(col);
     }, [col, handleAddColumn]);
 
+    const editColumnTitle = useCallback(() => {
+        setOpen(false);
+        setEditingTitle(true);
+    }, []);
+
+    const onColumnTitleUpdate = useCallback((newTitle: string) => {
+        onColumnTitleChange(col.set('caption', newTitle) as QueryColumn);
+    }, []);
```

Comment:
should the dep array here include `col` and `onColumnTitleChange`?

Reply:
Ah yes, of course.